--- origsrc/openexr_ctl-1.0.1/exr_ctl_exr/applyCtl.cpp	2007-07-12 22:24:37.000000000 -0500
+++ src/openexr_ctl-1.0.1/exr_ctl_exr/applyCtl.cpp	2011-07-11 21:36:33.249008600 -0500
@@ -64,6 +64,8 @@
 #include <iostream>
 #include <vector>
 #include <string>
+#include <cstdio>
+#include <cstdlib>
 
 using namespace std;
 using namespace Ctl;
--- origsrc/openexr_ctl-1.0.1/exr_ctl_exr/main.cpp	2007-07-25 01:02:11.000000000 -0500
+++ src/openexr_ctl-1.0.1/exr_ctl_exr/main.cpp	2011-07-11 21:37:02.715121100 -0500
@@ -59,6 +59,7 @@
 #include <iostream>
 #include <string>
 #include <exception>
+#include <cstdlib>
 #include <ImfFloatAttribute.h>
 #include <ImfVecAttribute.h>
 
--- origsrc/openexr_ctl-1.0.1/exrdpx/main.cpp	2007-07-25 01:28:17.000000000 -0500
+++ src/openexr_ctl-1.0.1/exrdpx/main.cpp	2011-07-11 21:35:11.196771900 -0500
@@ -60,6 +60,8 @@
 #include <iostream>
 #include <string>
 #include <exception>
+#include <cstdlib>
+#include <cstring>
 
 using namespace std;
 using namespace Imf;
